integration_tests/cork_test.go

```diff
@@ -125,8 +126,76 @@ func (s *IntegrationTestSuite) TestCork() {
 		s.Require().NoError(err)
 		s.Require().Equal(int64(0), count.Int64())
 
-		s.T().Logf("verify that contract exists in allowed addresses")
+		s.T().Log("verify that there are no allowed addresses on the new chain")
 		val := s.chain.validators[0]
+		kb, err := val.keyring()
+		s.Require().NoError(err)
+		clientCtx, err := s.chain.clientContext("tcp://localhost:26657", &kb, "val", val.keyInfo.GetAddress())
+		s.Require().NoError(err)
+		queryClient := types.NewQueryClient(clientCtx)
+		res, err := queryClient.QueryCellarIDs(context.Background(), &types.QueryCellarIDsRequest{})
+		s.Require().NoError(err)
+		s.Require().Empty(res.CellarIds)
+
+		s.T().Logf("create governance proposal to add counter contract")
+		orch := s.chain.orchestrators[0]
+		clientCtx, err = s.chain.clientContext("tcp://localhost:26657", orch.keyring, "orch", orch.keyInfo.GetAddress())
+		s.Require().NoError(err)
+
+		proposal := types.AddManagedCellarsProposal{
+			Title:       "add counter contract in test",
+			Description: "test description",
+			CellarIds: &types.CellarIDSet{
+				Ids: []string{counterContract.Hex()},
+			},
+		}
+		proposalMsg, err := govtypes.NewMsgSubmitProposal(
+			&proposal,
+			sdk.Coins{
+				{
+					Denom:  testDenom,
+					Amount: stakeAmount.Quo(sdk.NewInt(2)),
+				},
+			},
+			orch.keyInfo.GetAddress(),
+		)
+		s.Require().NoError(err, "unable to create governance proposal")
+
+		s.T().Log("submit proposal adding test cellar ID")
+		submitProposalResponse, err := s.chain.sendMsgs(*clientCtx, proposalMsg)
+		s.Require().NoError(err)
+		s.Require().Zero(submitProposalResponse.Code, "raw log: %s", submitProposalResponse.RawLog)
+
+		s.T().Log("check proposal was submitted correctly")
+		govQueryClient := govtypes.NewQueryClient(clientCtx)
+		proposalsQueryResponse, err := govQueryClient.Proposals(context.Background(), &govtypes.QueryProposalsRequest{})
+		s.Require().NoError(err)
+		s.Require().NotEmpty(proposalsQueryResponse.Proposals)
+		s.Require().Equal(uint64(1), proposalsQueryResponse.Proposals[0].ProposalId, "not proposal id 1")
+		s.Require().Equal(govtypes.StatusVotingPeriod, proposalsQueryResponse.Proposals[0].Status, "not proposal id 1")
```

Comment:
Did you intend for this error message to be the same as on line 174?

Reply:
Nope!